### PR TITLE
Fix IE11 split

### PR DESCRIPTION
**Pull request by BrennanConroy**
Turns out polyfilling isn't as straight forward as we thought. Basically the "this" wasn't passed through to our polyfill so we were just calling `Array.prototype.slice` which would return an empty array every time.

Going back to the original approach of manually checking for Uint8Array.

### CODE REVIEW

**Review by mikaelm12 — COMMENTED** [review]
Can we confirm that this will work before waiting for SauceLabs to fail?

File: src/SignalR/clients/ts/signalr/src/browser-index.ts [hunk 1]
```diff
@@ -16,8 +16,7 @@ if (!Uint8Array.prototype.indexOf) {
 }
 if (!Uint8Array.prototype.slice) {
     Object.defineProperty(Uint8Array.prototype, "slice", {
-        // wrap the slice in Uint8Array so it looks like a Uint8Array.slice call
-        value: (start?: number, end?: number) => new Uint8Array(Array.prototype.slice(start, end)),
+        value: Array.prototype.slice,
```
Comment (halter73) [inline src/SignalR/clients/ts/signalr/src/browser-index.ts]: > Basically the "this" wasn't passed through to our polyfill so we were just calling Array.prototype.slice which would return an empty array every time.

Why not just bind Array.prototype.slice to this?

```suggestion
        value: (start?: number, end?: number) => new Uint8Array(Array.prototype.slice.bind(this)(start, end))
```

**Review by halter73 — DISMISSED** [review]

**Comment by halter73** [comment]
> Can we confirm that this will work before waiting for SauceLabs to fail?

Agreed. It looks like the daily tests that caught this only take 17 minutes. @natemcmaster spoke to @mikaelm12 and me about a way to use Azure Pipelines run these tests on PR that opt into it. We should look into that.

#5114

File: src/SignalR/clients/ts/signalr/src/browser-index.ts (hunk 1 above)
Comment (BrennanConroy) [reply src/SignalR/clients/ts/signalr/src/browser-index.ts]: If you know how then please share, but I've tried many different things and none work.

**Comment by BrennanConroy** [comment]
I made sure this passes.

File: src/SignalR/clients/ts/signalr/src/browser-index.ts (hunk 1 above)
Comment (halter73) [reply src/SignalR/clients/ts/signalr/src/browser-index.ts]: ...

**Comment by mikaelm12** [comment]
> AspNetCore-SignalR-Daily-Tests is only available in the internal-only CI. You'll need to create a public version

Yeah, I was a bit confused on that. So his comment just re ran the regular valication  I'm guessing then?

**Review by mikaelm12 — APPROVED** [review]

**Pull request merged**